### PR TITLE
Marks Linux plugin_dependencies_test to be unflaky

### PR DESCRIPTION
**Pull request by fluttergithubbot**
The test has been passing for [50 consecutive runs](https://dashboards.corp.google.com/flutter_check_prod_test_flakiness_status_dashboard?p=BUILDER_NAME:%22Linux%20plugin_dependencies_test%22).
This test can be marked as unflaky.

### CODE REVIEW

**Review by jmagman — APPROVED** [review]
LGTM

**Comment by jmagman** [comment]
@chunhtai Should I manually merge this generated PR, or will cocoon merge it automatically at some point after a LGTM?

**Comment by chunhtai** [comment]
the current script will not merge the pr, although it may be a good feature to add into the script to auto merge once it is approved cc @keyonghan

**Pull request merged**
